client/state/selectors/can-current-user.js

```diff
@@ -17,12 +11,11 @@ import { isValidCapability } from 'calypso/state/current-user/selectors';
  * @param  {string}   capability Capability label
  * @returns {boolean}            Whether current user has capability
  */
-export const canCurrentUser = ( state, siteId, capability ) => {
-	if ( ! isValidCapability( state, siteId, capability ) ) {
-		return false;
+export default function canCurrentUser( state, siteId, capability ) {
+	const capabilities = state.currentUser.capabilities[ siteId ];
+	if ( ! capabilities ) {
+		return null;
 	}
 
-	return state.currentUser.capabilities[ siteId ][ capability ];
-};
-
-export default canCurrentUser;
+	return capabilities[ capability ] ?? false;
```

Comment:
This may be a bit of a micro-optimisation in case the build doesn't support `??`, but it could be better to use `||` instead here. Since we only expect boolean values, we can use the simpler syntax to forcefully convert any falsy values, instead of treating `null` and `undefined` differently.

Still, just a suggestion, perfectly valid to leave it as-is in case we want to support non-boolean values.